.npmrc

```diff
@@ -1,10 +0,0 @@
-# Make sure the default patterns are still included (https://pnpm.io/npmrc#public-hoist-pattern)
-public-hoist-pattern[]="*eslint*"
-public-hoist-pattern[]="*prettier*"
```

Comment:
These are no longer defaults in PNPM v10, but we don't even need them anymore since we depend directly on `pnpm` and `eslint` (and eslint flat config made them redundant too).